### DISCUSSION TITLE
Bed leveling using mesh

### DISCUSSION THREAD

**Discussion by roydonmalcolm** [Jyers/Marlin#755] [Q&A] [answered]
I'm new here and new to well almost all of this. i am eager to learn and being able to fix my issues and have consistent prints. To that end here is my question
I downloaded the latest firmware ( 1.3.2 ) was able to flash it to my printer ( ender 3 v2 and I have a CR10 coming I think I may have ot also but that's a whole new ball of wax as it has a bed level auto on it ) 
I then manually leveled my bed , went to the mesh leveling and used the z stepper to position it at each of the 25 points, clicked save, then I put the 420 line in my G-code Ultimaker Cura , selected a print and saved to my SD card I use for prints ( not the original with the other files or the one with the new flash its strictly has prints on it ) then I inserted into my printer and went to print but it was still all out and had to step the z axis till it started laying material. I thought it was supposed ot adjust at each of the points for me? did I miss a step? or a ton of steps? all I want is to print and have things actually STICK to my bed and lay down proper. Tired of one in 20 attempts the print actually works. Any helps or if there's a guide for my printer and this mesh thing ? thank anyone who has any info or help. I appreciate it

**Comment by msdossajo** [comment, marked as answer]
@roydonmalcolm after acquiring the 25 points (that are used to measure the bed flatness) you have to adjust the Z offset, don't forget to save it. This will be the starting point for every print.
